sprint0/sprint0/Weapons/FlameProjectile.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace sprint0
+{
+    class FlameProjectile : IProjectile
```

Comment:
what is flame projectile for? what do they look like? Are they for Ganon- because he has ganonFireballs

Reply:
FlameProjectile looks like the Flame block but is a projectile because that's what happens when Link uses the candle

Reply:
oh ok